model_analyzer/entrypoints/main.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
This should just be part of cli/ or at the top model_analyzer/ folder. What is the reason for a separate entrypoints folder?